### PR TITLE
add tolerations to adot exporter daemonset

### PR DESCRIPTION
**Pull request by jallen-frb**
**Description:**
* add tolerations to adot exporter daemonset
* default adotcollector tolerations values to empty list

**Link to tracking Issue:** N/A

**Testing:** Was not tested

**Documentation:** No documentation needed

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: charts/adot-exporter-for-eks-on-ec2/templates/adot-collector/daemonset.yaml [hunk 1]
```diff
@@ -33,4 +33,6 @@ spec:
       volumes:
         {{- toYaml .Values.adotCollector.daemonSet.volumes | nindent 10 }}
       serviceAccountName: {{ .Values.adotCollector.daemonSet.serviceAccount.name  }}
+      tolerations:
+        {{- toYaml .Values.adotCollector.daemonSet.tolerations | nindent 8 }}
```
Comment (Aneurysm9) [inline charts/adot-exporter-for-eks-on-ec2/templates/adot-collector/daemonset.yaml:37]:
```suggestion
        {{- toYaml .Values.adotCollector.daemonSet.tolerations | nindent 10 }}
```

I think `10` is the correct value here, based on what is used above for `volumes`.

**Review by mhausenblas — APPROVED** [review]

**Comment by RogerWatkins-Anaplan** [comment]
I noticed the adot-collector-daemonset is not scheduling on all our cluster nodes so 👍 to this!

**Comment by RogerWatkins-Anaplan** [comment]
any plans to merge this ? 🙏

**Comment by vasireddy99** [comment]
@jallen-frb Thanks for the PR, can you please add any testing result, just spinning up and the log would be good enough.

**Comment by s33kers** [comment]
A bit too late but missing update in `values.schema.json` for new value support.

Also, when it's going to be published to repo? or version bump missing? I was fighting demons today till found out that package in https://aws-observability.github.io/aws-otel-helm-charts isn't matching main branch source code..

**Comment by vasireddy99** [comment]
> A bit too late but missing update in `values.schema.json` for new value support.
> 
> Also, when it's going to be published to repo? or version bump missing? I was fighting demons today till found out that package in https://aws-observability.github.io/aws-otel-helm-charts isn't matching main branch source code..
> 
> @vasireddy99 @mhausenblas @jallen-frb

Created this PR - https://github.com/aws-observability/aws-otel-helm-charts/pull/105 for values.schema.json

File: charts/adot-exporter-for-eks-on-ec2/templates/adot-collector/daemonset.yaml (hunk 1 above)
Comment (bryan-aguilar) [reply charts/adot-exporter-for-eks-on-ec2/templates/adot-collector/daemonset.yaml:37]: I have confirmed that level 8 indentation iworks by using `helm template` and then applying the manifest generated
values
```
    tolerations:
      - key: "key1"
        operator: "Exists"
        effect: "NoSchedule"
```
manifest after template
```
          - hostPath:
              path: /dev/disk/
            name: devdisk
      serviceAccountName: adot-collector-sa
      tolerations:
        - effect: NoSchedule
          key: key1
          operator: Exists
```
daemonset spec
```
│ Tolerations:                 key1:NoSchedule op=Exists                                                                                                    │
│                              node.kubernetes.io/disk-pressure:NoSchedule op=Exists                                                                        │
│                              node.kubernetes.io/memory-pressure:NoSchedule op=Exists                                                                      │
│                              node.kubernetes.io/not-ready:NoExecute op=Exists                                                                             │
│                              node.kubernetes.io/pid-pressure:NoSchedule op=Exists                                                                         │
│                              node.kubernetes.io/unreachable:NoExecute op=Exists                                                                           │
│                              node.kubernetes.io/unschedulable:NoSchedule op=Exists                                                                        │
│ Events:                                                                              
```

**Pull request merged**